src/IB/IBFEMethod.cpp

```diff
@@ -1612,6 +1679,7 @@ IBFEMethod::addWorkloadEstimate(Pointer<PatchHierarchy<NDIM> > hierarchy, const
         std::vector<std::size_t> dofs_per_processor(n_processes);
         for (unsigned int part = 0; part < d_num_parts; ++part)
         {
+            if (!d_part_is_active[part]) continue;
```

Comment:
Let's not include this guard --- I think that I would like for all computations to proceed "as usual" for inactivated parts, except that at the end of the day, forces are not spread and velocity is not interpolated.